registry/storage/driver/s3-aws/s3.go

```diff
@@ -230,6 +243,8 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		fmt.Sprint(rootDirectory),
 		storageClass,
 		fmt.Sprint(userAgent),
+		fmt.Sprint(backend),
```

Comment:
@arschles fmt.Sprint(rootDirectory)  is present in the existing docker repo and hence i continued on using it.Is there any harm in keeping it?